packages/web-frontend/src/containers/OverlayDiv/index.js

```diff
@@ -70,6 +76,9 @@ export const OverlayDiv = ({
 
   const components = {
     [LANDING]: () => <LandingPage isUserLoggedIn={isUserLoggedIn} transition />,
+    [VIEW_PROJECTS]: () => (
+      <LandingPage isUserLoggedIn={isUserLoggedIn} isViewingProjects transition />
+    ),
```

Comment:
Is this a tad awkward? Felt like the simplest way I could re-route the non-logged in user to the 'View projects' page...

Reply:
Good idea! Will do 👍